core/invoker/src/main/scala/whisk/core/containerpool/ContainerProxy.scala

```diff
@@ -382,8 +384,8 @@ object ContainerProxy {
      * @param suffix the container name's suffix
      * @return a unique container name
      */
-    def containerName(prefix: String, suffix: String) =
-        s"wsk_${containerCount.next()}_${prefix}_${suffix}".replaceAll("[^a-zA-Z0-9_]", "")
+    def containerName(instance: InstanceId, prefix: String, suffix: String) =
+        s"wsk${instance.toInt.toString}_${containerCount.next()}_${prefix}_${suffix}".replaceAll("[^a-zA-Z0-9_]", "")
```

Comment:
Please omit the `toString`, it's not needed inside this string interpolation, the integer is fine.